docs/usage.md

```diff
@@ -404,8 +404,8 @@ Just like [render_node()](#render-elements-without-a-parent-orphans), there is
 without a parent:
 
 ```pycon
->>> from htpy import li, iter_node
->>> for chunk in iter_node([li["a"], li["b"]]):
+>>> from htpy import li, Fragment
+>>> for chunk in fragment[[li["a"], li["b"]]]:
```

Comment:
Isn't there one level of brackets more than needed?

Reply:
good catch. it is fixed now!